### PR TITLE
Bugfix/hacked stuff to make it work

### PR DESCRIPTION
**Pull request by cecemel**
I hacked some things to make it work. Don't merge the PR, just things we need to take into account and fix properly

### CODE REVIEW

**Review by cecemel — COMMENTED** [review]
The change to TEMP_GRAPH_SCANNING_INTERVAL may be ignored.

File: app.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ export async function encapsulatedScanAndProcess(processDeletes) {
  * happens once on startup of the service. The reason for this being on a timer
  * is that it can be delayed if needed.
  */
-setTimeout(encapsulatedScanAndProcess, 0);
+setTimeout(encapsulatedScanAndProcess, TEMP_GRAPH_SCANNING_INTERVAL);
```
Comment (cecemel) [inline app.js:55]: Nevermind this, this was just euum dumb

**Comment by benjay10** [comment]
Converted to draft. Issues will be solved in other PR's and this PR can then be closed.

**Comment by benjay10** [comment]
This PR will be closed. Fixes for these issues can be found here:

* For the UUIDs
    https://github.com/lblod/app-lblod-harvester/pull/28
    https://github.com/lblod/import-with-sameas-service/pull/9
* For the graph postfixes
    https://github.com/lblod/worship-positions-graph-dispatcher-service-loket/pull/5
* For the mu-call-scope-id
    https://github.com/lblod/worship-positions-graph-dispatcher-service-loket/pull/6

**Pull request closed without merging**